docs/docs/release-notes.md

```diff
@@ -7,8 +7,12 @@
 2. EPID attestation protocol has been deprecated, and it will be removed in an upcoming release. You should use DCAP instead.
 3. For security reasons, the way [enclave constraints](api/-conclave%20-core/com.r3.conclave.common/-enclave-constraint/index.html)
    are evaluated has changed slightly. From now on, if a client sets the security level to `INSECURE`, it will not pass
-   for `STALE` or `SECURE` enclaves. This is to prevent the client from accidently communicating with a production
+   for `STALE` or `SECURE` enclaves. This is to prevent the client from accidentally communicating with a production
    enclave during development or testing.
+4. The configuration field `enclaveSize` can be used to set the enclave size when the runtime is set to `gramine`.  The 
```

Comment:
I wouldn't mention this as a separate point since it's part of the bigger Gramine feature

Reply:
Please review PR https://github.com/R3Conclave/conclave-core-sdk/pull/155.